vagrant-pxe-harvester/ansible/roles/harvester/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: create Harvester config dir
+  file:
+    path: /var/www/harvester
+    state: directory
+
+- name: download Harvester media
+  get_url:
+    url: https://releases.rancher.com/harvester/{{ settings['harvester_version'] }}/{{ item }}
```

Comment:
can we set the base URL configurable or use a local image as it will speed up the download speed a lot, thanks.

Reply:
Sure, I'll make the URLs configurable.